RecoTracker/TkSeedGenerator/plugins/JetCoreMCtruthSeedGenerator.h

```diff
@@ -0,0 +1,193 @@
+#ifndef RecoTracker_TkSeedGenerator_JetCoreMCtruthSeedGenerator_H
+#define RecoTracker_TkSeedGenerator_JetCoreMCtruthSeedGenerator_H
+
+#define jetDimX 30  //pixel dimension of NN window on layer2
+#define jetDimY 30  //pixel dimension of NN window on layer2
+#define Nlayer 4    //Number of layer used in DeepCore
+#define Nover 3     //Max number of tracks recorded per pixel
+#define Npar 5      //Number of track parameter
```

Comment:
as mentioned, use static constexpr in the class def

Reply:
ok